certificate.proto

```diff
@@ -23,9 +23,21 @@ message CertificateMethod {
     }
 }
 
+message RSAPayload {
+    int32 key_size = 1;
+    int32 public_exponent = 2;
+}
+
 message NewCertificatePayload {
```

Comment:
Let's rename Certificate to PublicKey. 
NewPublicKeyPayload, 
RevokePublicKeyPayload .... 
@eugene-babichenko agree with that ?

Reply:
Yes.